distsql/request_builder_test.go

```diff
@@ -42,6 +45,23 @@ func TestT(t *testing.T) {
 var _ = Suite(&testSuite{})
 
 type testSuite struct {
+	sctx sessionctx.Context
+}
+
+func (s *testSuite) SetUpSuite(c *C) {
+	ctx := mock.NewContext()
+	ctx.Store = &mock.Store{&mock.Client{&mockResponse{}}}
+	s.sctx = ctx
+}
+
+func (s *testSuite) TearDownSuite(c *C) {
+}
+
+func (s *testSuite) SetUpTest(c *C) {
+	testleak.BeforeTest()
```

Comment:
Where is `testleak.AfterTest()()` called?

Reply:
added in `TearDownTest`